charts/scalardb-cluster/templates/scalardb-cluster/deployment.yaml

```diff
@@ -66,9 +66,7 @@ spec:
                 - -addr=localhost:60053
                 {{- if .Values.scalardbCluster.tls.enabled }}
                 - -tls
-                {{- if .Values.scalardbCluster.tls.caRootCertSecret }}
-                - -tls-ca-cert=/tls/certs/ca-root-cert.pem
-                {{- end }}
+                - -tls-ca-cert=/tls/scalardb-cluster/certs/ca.crt
```

Comment:
When you enable TLS, the `-tls-ca-cert` option is mandatory. In other words, `{{- if .Values.scalardbCluster.tls.caRootCertSecret }}` does not make sense. So, I removed that unnecessary condition.